packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/project/dto/CreateProjectSuccessPayload.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.project.dto;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.core.api.IPayload;
+
+import jakarta.validation.constraints.NotNull;
+
+/**
+ * The input used to create projects.
+ *
+ * @author wpiers
+ */
+public record CreateProjectSuccessPayload(@NotNull UUID id, @NotNull ProjectDTO project) implements IPayload {
+    public CreateProjectSuccessPayload {
+        Objects.requireNonNull(id);
+        Objects.requireNonNull(project);
```

Comment:
Right now yes because there's nothing performing this `@NotNull` validation but I can remove them and wait for proper support of the validation when the time will be right